app/src/routes/homeRoutes.tsx

```diff
@@ -0,0 +1,14 @@
+import RootComponent from "components/redirects/RootComponent";
```

Comment:
- app/src/features/home
    - routes.tsx


We can later on move other components of HomePage here too from `components` folder. For now this will be a empty folder

Reply:
This is independent of current changes. We can do this later too